fusion/tsdf.py

```diff
@@ -0,0 +1,389 @@
+# Copyright (c) 2018 Andy Zeng
+
+# Library imports
+import numpy as np
+import time
+from numba import njit, prange
+from skimage import measure
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchvision import datasets, transforms
+
+
+try:
```

Comment:
Please, reformat all source files to comply to PEP-8 specification. E.g. PyCharm (even the free community edition) can do this automatically for you.

Reply:
Yes I will will do it in the next PR. Currently I am focusing on integrating  Warp field pipeline. I will refactor the code to PEP-8 

Thanks for the review